extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/stream/YoutubeStreamExtractorDefaultTest.java

```diff
@@ -407,14 +407,6 @@ public static void setUp() throws Exception {
         @Nullable @Override public String expectedTextualUploadDate() { return "2019-06-12"; }
         @Override public long expectedLikeCountAtLeast() { return 70000; }
         @Override public long expectedDislikeCountAtLeast() { return -1; }
-        @Override public List<MetaInfo> expectedMetaInfo() throws MalformedURLException {
-            return Collections.singletonList(new MetaInfo(
-                    EMPTY_STRING,
-                    new Description("Funk is a German public broadcast service.", Description.PLAIN_TEXT),
-                    Collections.singletonList(new URL("https://de.wikipedia.org/wiki/Funk_(Medienangebot)?wprov=yicw1")),
-                    Collections.singletonList("Wikipedia (German)")
-            ));
-        }
```

Comment:
No other tests in this file seem to be overriding `expectedMetaInfo`. I actually suspect the `YoutubeStreamExtractor` currently can't extract meta info -- it is contained within the `videoPrimaryInfoRenderer` and `videoSecondaryInfoRenderer` objects rather than there the current `YoutubeParsingHelper.getMetaInfo()` is looking for it. How would we better address this?